src/firefly/js/data/form/PositionFieldDef.js

```diff
@@ -190,8 +190,8 @@ var makePositionFieldDef= function(properties) {
 
 
 
-        var lonStr = numeral(wp.getLon()).format('#.0[00000]');
-        var latStr = numeral(wp.getLat()).format('#.0[00000]');
+        var lonStr = sprintf('%6.5f',wp.getLon());
+        var latStr = sprintf('%6.5f',wp.getLat());
```

Comment:
I noticed you're  using `width.precision`.  `width` is the minimum number of characters to output.  This is usually use for padding.  In your case, it makes no difference since the results will always be more than 6 characters, meaning `%.5f` would have suffice.